src/pres/MainContainer.js

```diff
@@ -49,6 +49,7 @@ export default class MainContainer extends React.Component {
         gamesProcessed:0,
         openingGraph:new OpeningGraph(selectedVariant),
         settings:{
+          variant:selectedVariant,
```

Comment:
I think the variant in state just a few lines below and replace the instances with `state.settings.variant`? Its better to not have it duplicated